tests/kit/model_zoo/transformers/chatglm2_6b/MODEL_LICENSE

```diff
@@ -0,0 +1,33 @@
+The ChatGLM2-6B License
```

Comment:
You don't need to put the license here, you can just mention this license at the top of the `modeling_chatglm.py`.

Reply:
ok